packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -38,8 +40,8 @@ class ValidationDetailView extends StatelessWidget {
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
               Text(
-                'This tool assistants helps you diagnose Universal Links, App Links,'
-                ' and Custom Schemes in your app. Web check are done for the web association'
+                'This tool assistants helps you diagnose App Links in your app. '
```

Comment:
wording nit: "This tool helps you diagnose issues with App Links in your application."